extensions/gitea/package.json

```
@@ -0,0 +1,158 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "gitea",
+  "title": "Gitea",
+  "description": "Manage your Gitea instance",
+  "icon": "extension-icon.png",
+  "author": "maantje",
+  "categories": [
+    "Productivity",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "create-issue",
+      "title": "Create Issue",
```

```suggestion
      "title": "Create Issue",
      "subtitle": "Gitea",
```
I would do this for all the commands

---

Hey @pernielsentikaer,

Thank you for initial review. I will add the subtitles, and create a screencast of the flow soon.